features/facebookImport/src/views/report/details.css

```diff
@@ -4,31 +4,17 @@
 
 .report-details .report-card {
     max-width: var(--max-width);
-    max-height: 440px;
     margin: 0 auto 12px auto;
     padding: 12px 20px 20px 20px;
     border-radius: 8px;
     background-color: var(--background-light);
     color: var(--text-dark);
-    overflow: scroll;
 }
 
-.report-details .report-card-scrolling {
-    max-width: var(--max-width);
-    position:relative;
-    margin: auto;
-}
-
-.report-details .report-card-scrolling::after {
-    content: "";
-    width: 100%;
-    height: 30px;
-    background: linear-gradient(#ffffff00 0%, #ffffff);
-    position: absolute;
-    bottom: 12px;
-    left: 0;
-    border-radius: 8px;
-
+.report-details .report-card table {
+    display: block;
+    overflow-x: scroll;
+    overflow-y: hidden;
```

Comment:
Hm, wouldn't it be `visible`? That said, if the `table` doesn't have a height, it probably can't overflow vertically, so might work the same if you leave this one out?

Reply:
Do you mean `overflow-x: visible`? I've tried it and in this case, it doesn't make the content visible. If we want to show the content, the overflow-x could be `scroll` or `auto`. Or we can use the `word-break: break-all` if we don't want any kind of scrolling, but the tables don't look really good.

Regarding the `overflow-y`, you're right, that one is not necessary so I'll remove it.